Model/AdyenThreeDS2Process.php

```diff
@@ -145,13 +145,6 @@ public function initiate($payload)
         if($result['resultCode'] != 'Authorised') {
             $this->checkoutSession->restoreQuote();
 
-            // Always cancel the order if the paymenth has failed
-            if (!$order->canCancel()) {
-                $order->setState(\Magento\Sales\Model\Order::STATE_NEW);
-            }
-
-            $order->cancel()->save();
-
```

Comment:
This is a synchronous response so the cancellation is valid

Reply:
Restoring this cancellation because it isn't redirect